plugins/lua/sort/places.lua

```diff
@@ -112,8 +112,36 @@ local function get_location_search_key(zone)
 end
 
 local function get_stockpile_search_key(stockpile)
-    if #stockpile.name ~= 0 then return stockpile.name
-    else return ('Stockpile #%s'):format(stockpile.stockpile_number) end
+    local result = {}
+    if #stockpile.name ~= 0 then table.insert(result, stockpile.name) end
+
+    local flags = stockpile.settings.flags
+    local flag_names = {
```

Comment:
could you move this table outside of the function? since it's static, there's no reason to reinitialize it every time the function is called.

Reply:
Great point. Fixed it.